<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskDialog</class>
 <widget class="QDialog" name="MaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaskDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="infoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="MaskView" name="graphicsView">
     <property name="toolTip">
      <string>Save mask</string>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="toolTip">
        <string>Edit</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resources/resources.qrc">
         <normaloff>:/icons_64/pencil.png</normaloff>:/icons_64/pencil.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="toolTip">
        <string>Add</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resources/resources.qrc">
         <normaloff>:/icons_64/add.png</normaloff>:/icons_64/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="toolTip">
        <string>Remove</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resources/resources.qrc">
         <normaloff>:/icons_64/remove.png</normaloff>:/icons_64/remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>Clear All</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resources/resources.qrc">
         <normaloff>:/icons_64/clear.png</normaloff>:/icons_64/clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MaskView</class>
   <extends>QGraphicsView</extends>
   <header>MaskView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskDialog</receiver>
   <slot>onEditClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskDialog</receiver>
   <slot>onAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskDialog</receiver>
   <slot>onClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskDialog</receiver>
   <slot>onRemoveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MaskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MaskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onEditClicked()</slot>
  <slot>onAddClicked()</slot>
  <slot>onRemoveClicked()</slot>
  <slot>onClearClicked()</slot>
 </slots>
</ui>
